extensions/wikia/Email/Email.i18n.php

```diff
@@ -8,7 +8,6 @@
 
 	// General errors
 	'emailext-error-restricted-controller' => 'Access to this controller is restricted',
-	'emailext-error-noname' => 'Required username has been left empty',
```

Comment:
Ah, Garth, Noelle, and I actually had a conversation about these yesterday since Noelle starting translating them.

Garth actually asked that we do have these translated if it wasn't too much trouble (Noelle said it wouldn't be). We agreed that right now only developers should see them, but just in case there's a use case for bubbling these up in the future it wouldn't hurt to have them translated. So we can leave these as is.